src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -317,7 +317,7 @@ export default class ResultsViewPage extends React.Component<
                         <MSKTab
                             key={10}
                             id={ResultsViewTab.COMPARISON}
-                            linkText={'Comparison'}
+                            linkText={'Comparison/Survival'}
```

Comment:
should the tab name only show survival if there is survival data? (see the `hide` condition for SURVIVAL_REDIRECT)